service/src/main/java/uk/gov/hmcts/reform/fpl/service/OthersService.java

```diff
@@ -39,4 +44,136 @@ private boolean otherExists(Others others) {
         return others != null && (others.getFirstOther() != null || others.getAdditionalOthers() != null);
     }
 
+    public Others modifyHiddenValues(Others others) {
+        final List<Element<Other>> othersList = new ArrayList<>();
+        Element<Other> firstOther;
+
+        if (others.getFirstOther().containsConfidentialDetails()) {
+            firstOther = Element.<Other>builder()
+                .value(others
+                    .getFirstOther()
+                    .toBuilder()
+                    .address(null)
+                    .telephone(null)
+                    .build())
+                .build();
+        } else {
+            firstOther = Element.<Other>builder()
+                .value(others.getFirstOther()).build();
+        }
+
+        others.getAdditionalOthers().stream().forEach(additionalOther -> {
+            if (additionalOther.getValue().containsConfidentialDetails()) {
+                othersList.add(Element.<Other>builder()
+                    .id(additionalOther.getId())
+                    .value(additionalOther.getValue().toBuilder().address(null).telephone(null).build())
+                    .build());
+            } else {
+                othersList.add(Element.<Other>builder()
```

Comment:
othersList.add(element(additionalOther.getId(), additionalOther.getValue()), maybe even just additional other passed in?